charts/sxt-node-chart/values.yaml

```diff
@@ -30,6 +30,11 @@ sxt:
   p2pService: true
   port: 30333
   rpcPort: 9944
+  rpcRate: 250
+  rpcMethods: "safe"
+  rpcMaxConnections: 10000
+  rpcMaxRequestSize: 1024
+  rpcMaxResponseSize: 1024
```

Comment:
looks fine, but if these fields are all related to rpc, would it be best to group them kinda like it was done for `flightSQL`?

We'd need to change `isRpcNode` to be something like `.sxt.rpc.enabled` and then all these are under `.sxt.rpc` as well. It's okay if we don't, but just a thought.